lib/platform-api/requests.js

```diff
@@ -55,6 +55,36 @@ const getSchema = async (schemaName) => {
   }
 }
 
+const holdingById = async (id) => {
+  try {
+    const client = await platformApi.client()
+    const resp = await client.get(`holdings/${id}`)
+    if (resp) {
+      return resp
+    } else {
+      logger.warn(`Warning: holding not found: ${id}`)
+      return null
+    }
+  } catch (e) {
+    logger.error('PlatformApi#holdingById error:\n', e)
+  }
+}
+
+const itemById = async (nyplSource, id) => {
+  try {
+    const client = await platformApi.client()
+    const resp = await client.get(`items/${nyplSource}/${id}`)
+    if (isValidResponse(resp)) {
+      return resp.data
+    } else {
+      logger.warn(`Warning: item not found: ${nyplSource}/${id}`)
+      return null
+    }
+  } catch (e) {
+    logger.error('PlatformApi#itemById error:\n', e)
+  }
+}
+
```

Comment:
These could probably be refactored since they share a lot of logic but it doesn't seem like a big deal